requirements.txt

```diff
@@ -11,6 +11,7 @@ backports.ssl-match-hostname==3.5.0.1  # via apache-libcloud
 caslib.py==2.2.2          # via django-cyverse-auth
 cffi==1.10.0              # via cryptography
 contextlib2==0.5.5        # via raven
+django-cors-headers==1.3.1
```

Comment:
I think you might have spoke to Connor, but you'll want to define this and the `whitenoise` module additions in `requirement.in` and you'll want to generate the _superset_ `dev_requirements.in`. 

See Connor's explanation: 
- https://github.com/cyverse/troposphere/blob/master/REQUIREMENTS.md